src/Component/Calculator.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/prop-types, react/prefer-stateless-function */
+import React, { Component } from 'react';
+import './Calculator.css';
+
+class Calculator extends Component {
+  render() {
+    const tope = this.props;
+    return (
+      <div className="btn-cont d-flex j-center a-center">
+        <div className="result d-flex a-center">
+          <p>
+            {tope.result}
+          </p>
+        </div>
+        <div className="btn-container">
+          <div className="first-row">
+            <button className="button" data-id="AC" type="button">AC</button>
+            <button className="button" data-id="oper" type="button">+/-</button>
+            <button className="button" data-id="oper" type="button">%</button>
+            <button className="button" data-id="oper" type="button"> / </button>
+          </div>
+          <div className="second-row">
+            <button className="button" data-id="number" type="button">7</button>
+            <button className="button" data-id="number" type="button">8</button>
+            <button className="button" data-id="number" type="button">9</button>
+            <button className="button" data-id="oper" type="button">  x </button>
+          </div>
+          <div className="third-row">
+            <button className="button" data-id="number" type="button">4</button>
+            <button className="button" data-id="number" type="button">5</button>
+            <button className="button" data-id="number" type="button">6</button>
+            <button className="button" data-id="oper" type="button"> - </button>
+          </div>
+          <div className="fourth-row">
+            <button className="button" data-id="number" type="button">1</button>
+            <button className="button" data-id="number" type="button">2</button>
+            <button className="button" data-id="number" type="button">3</button>
+            <button className="button" data-id="oper" type="button"> + </button>
+          </div>
+          <div className="fifth-row">
+            <button className="button zero" data-id="number" type="button">0</button>
+            <button className="button" data-id="number" type="button"> . </button>
+            <button className="button" data-id="number" type="button"> = </button>
+          </div>
+        </div>
```

Comment:
- [ ] The orange _operation buttons_ do not follow the order given in the [template design](https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/raw/master/react_redux/math_magicians/images/calculator.png). Kindly ensure that the component **matches** the given design.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/143015703-99bc27f4-01d7-477b-b81c-80c0168d1c1a.png)


**The original design:**
![image](https://user-images.githubusercontent.com/12745474/143015782-26d30fa3-a576-4e4b-b169-cfb8998c4279.png)

Reply:
Thanks for your review @rloterh. Please just to be clear, is this issue with the division symbol I used. Because aside from that, everything seems in order to me. Please can you clarify?

Reply:
Hi @mckent05

After a second check, you are right! Sorry, I missed this aspect during my review. Kindly know that this was not a deliberate attempt!

Cheers, and Happy coding! 😃